agent-core/src/main/java/com/newrelic/agent/android/FeatureFlag.java

```diff
@@ -55,5 +56,6 @@ public static void resetFeatures() {
         enableFeature(HandledExceptions);
         enableFeature(DistributedTracing);
         enableFeature(AppStartMetrics);
+        enableFeature(OfflineStorage);
```

Comment:
Should not be enabled by default.

Reply:
@cthomas-newrelic any new feature should be enabled by default and need to give capability to customer to disable by flag.

Reply:
No, just the opposite. New feature must be opted-in by the customer. This limits the blast radius from any unintended consequence. It is then enabled by default after a reasonable cooling period. This is how we've always rolled out new features.

Reply:
@cthomas-newrelic  agree with you here, just confused with the docs I received recently.